modules/client/examples.md

```diff
@@ -63,15 +63,15 @@ import {
   Client,
   Context,
   DaoCreationSteps,
+  DaoMetadata,
   GasFeeEstimation,
-  ICreateParams,
 } from "@aragon/sdk-client";
-import { IMetadata } from "../../src";
+import { CreateDaoParams } from "../../src/interfaces";
```

Comment:
Shouldn't all example code snippets be imported from the NPM package? I would not show these local, repo-dependent paths

Reply:
Yes, my bad